ui/components/multichain/asset-picker-amount/asset-picker-modal/index.scss

```diff
@@ -89,29 +89,6 @@
       overflow: auto;
     }
 
-    &__main-view &__tab {
-      @include design-system.H6;
-
-      flex-grow: 1;
-      color: var(--color-icon-default);
-      font-weight: 500;
-    }
-
-    &__main-view &__tab--active {
-      color: var(--color-primary-default);
-    }
-
-    &__main-view &__tab button {
-      padding: 8px;
-    }
-
-    &__tabs {
-      display: grid;
-      grid-template-columns: 1fr 1fr;
-      border-bottom: 0;
-      padding-inline: 16px;
-    }
```

Comment:
These styles aren't necessary and have been removed. 
- `modal-tab__main-view__tab`
- `modal-tab__main-view__tab--active`
- `modal-tab__main-view__tab button`
- `modal-tab__tabs`

<img width="302" height="251" alt="Screenshot 2025-09-18 at 11 36 47 AM" src="https://github.com/user-attachments/assets/e1bf57d3-3536-49de-b3b3-2132ba042c06" /><img width="304" height="262" alt="Screenshot 2025-09-18 at 11 36 58 AM" src="https://github.com/user-attachments/assets/1bfd0368-9a7c-4b25-a8bf-6f6162de6c79" />